jhipster-framework/src/main/java/tech/jhipster/config/apidoc/SpringfoxAutoConfiguration.java

```diff
@@ -135,16 +135,13 @@ public JHipsterSpringfoxCustomizer jHipsterSpringfoxCustomizer() {
      * Springfox configuration for the management endpoints (actuator) OpenAPI docs.
      *
      * @param appName               the application name
-     * @param managementContextPath the path to access management endpoints
      * @return the Springfox configuration
      */
     @Bean
     @ConditionalOnClass(name = "org.springframework.boot.actuate.autoconfigure.web.server.ManagementServerProperties")
-    @ConditionalOnProperty("management.endpoints.web.base-path")
     @ConditionalOnExpression("'${management.endpoints.web.base-path}'.length() > 0")
```

Comment:
Should be removed also, no?

Reply:
@DanielFran and @PierreBesson On the contrary. Even the
```java
@ConditionalOnProperty("management.endpoints.web.base-path")
```
should stay in my opinion as this property hasn't been touched.
Fixed that here 0bc2cb9. Feel free to cherry pick.

Reply:
@DanielFran Correct. But I think it does not make sense to initialize the Docket for the management apis if `management.endpoints.web.base-path` has not been set in the first place. That's what the `@ConditionalOnProperty("management.endpoints.web.base-path")` was originally intended for, I think. Should have made that clearer in my first comment. Am I missing something?

Reply:
@clocken The property have by default `"/management/.*"`.
User can use the property to rename the path and include or not the context path (example: `"/admin/.*"` or `"/mycontext/admin/.*"`).
So it not depends anymore from the property `management.endpoints.web.base-path`

Reply:
@DanielFran Ok, now I see. Was missing that the `managementContextPath` was set to that property value before, sorry. Then I agree with you that the second line `@ConditionalOnExpression("'${management.endpoints.web.base-path}'.length() > 0")` should be removed also.

Reply:
@PierreBesson Done in a59a29c

Reply:
@PierreBesson Can you confirm if it should be removed or not?

Reply:
Yes it should be removed ! Sorry for the super late reply.